app/controllers/items_controller.rb

```diff
@@ -62,13 +62,14 @@ def destroy
   end
 
   private
-    # Use callbacks to share common setup or constraints between actions.
-    def set_item
-      @item = Item.find(params[:id])
-    end
 
-    # Never trust parameters from the scary internet, only allow the white list through.
-    def item_params
-      params.require(:item).permit(:name, :price, :description)
-    end
+  # Use callbacks to share common setup or constraints between actions.
+  def set_item
```

Comment:
__[reek]__
* [UnusedPrivateMethod] has the unused private instance method `set_item` [:ref](https://github.com/troessner/reek/blob/master/docs/Unused-Private-Method.md)